docs/Limitations.md

```diff
@@ -216,6 +218,16 @@ Equivalent functionality can be achieved with the newer docker networking comman
 See more documentation at
 [docs.docker.com](https://docs.docker.com/engine/userguide/networking/default_network/dockerlinks/).
 
+## Storage limitations
+
+### Subpaths
+
+Kubernetes `subPath` is not supported by Kata at the moment.
```

Comment:
I think it was @jodh-intel who once said me to avoid calling the project simply "Kata" on user-facing text. Instead, "Kata Containers".

Reply:
Good point. Fixed.